### PR TITLE
Optionally load and use Unicode::UTF8

### PR DESCRIPTION
**Pull request by xsawyerx**
Data needed. :)

### CODE REVIEW

File: lib/Dancer2/Core/Request.pm [hunk 1]
```diff
@@ -37,6 +37,16 @@ sub $_ { \$_[0]->env->{ 'HTTP_' . ( uc "$_" ) } }
 1;
 _EVAL
 
+eval {
+    require_module('Unicode::UTF8');
+    1;
```
Comment (SysPete) [inline lib/Dancer2/Core/Request.pm]: what is this doing here?

Comment (xsawyerx) [reply lib/Dancer2/Core/Request.pm]: Resolved.

**Comment by xsawyerx** [comment]
I've rebased this PR and fixed up what was necessary.

My benchmarks showed a minimal improvement, but hey, maybe it's enough or maybe my benchmark is not accurate enough.

**Comment by cromedome** [comment]
You might wish to add `Unicode::UTF8` as recommended in `cpanfile` (also, in `share/skel/cpanfile`).

I will comment more fully soon!

**Comment by cromedome** [comment]
Merged with one small change to `share/skel/cpanfile`. Thanks!

**Pull request closed without merging**